lib/media/ver10/set_video_encoder_configuration.ex

```diff
@@ -82,13 +82,16 @@ defmodule Onvif.Media.Ver10.SetVideoEncoderConfiguration do
   end
 
   def response(xml_response_body) do
-    xml_response_body
-    |> parse(namespace_conformant: true, quiet: true)
-    |> xpath(
-      ~x"//s:Envelope/s:Body/trt:SetVideoEncoderConfigurationResponse/text()"s
-      |> add_namespace("s", "http://www.w3.org/2003/05/soap-envelope")
-      |> add_namespace("trt", "http://www.onvif.org/ver10/media/wsdl")
-      |> add_namespace("tt", "http://www.onvif.org/ver10/schema")
-    )
+    res =
+      xml_response_body
+      |> parse(namespace_conformant: true, quiet: true)
+      |> xpath(
+        ~x"//s:Envelope/s:Body/trt:SetVideoEncoderConfigurationResponse/text()"s
+        |> add_namespace("s", "http://www.w3.org/2003/05/soap-envelope")
+        |> add_namespace("trt", "http://www.onvif.org/ver10/media/wsdl")
+        |> add_namespace("tt", "http://www.onvif.org/ver10/schema")
+      )
```

Comment:
can we parse this as a VideoEncoderConfiguration? I haven't looked at the XML so I don't know what it looks like

Reply:
The response XML is as below. Once parsed it will be just an empty string so I think we can have the parsing logic here in the module.
```
<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://www.w3.org/2003/05/soap-envelope"
xmlns:trt="http://www.onvif.org/ver10/media/wsdl">
<SOAP-ENV:Body>
<trt:SetVideoEncoderConfigurationResponse>
</trt:SetVideoEncoderConfigurationResponse>
</SOAP-ENV:Body>
</SOAP-ENV:Envelope>
```